dogs/src/app/components/breed-page/breed-page.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { Subscription } from "rxjs";
+import { ImageDTO } from "src/app/models/image.model";
+import { DogService } from "src/app/services/dog.service";
+
+@Component({
+	selector: "breed-page",
+	templateUrl: "./breed-page.component.html",
+	styleUrls: ["./breed-page.component.css"]
+})
+export class BreedPageComponent implements OnInit {
+	public breed?: string;
+	public image?: string;
+	public errorMessage?: string;
+	public subscriptionImage?: Subscription;
+	public subscriptionSubBreeds?: Subscription;
```

Comment:
you can get rid of these 2. If you want to unsubscribe from an observable you need to store it in a Subscription